recipes/sksparse/conda_build_config.yaml

```diff
@@ -0,0 +1,13 @@
+python:
+  - 3.6
+  - 3.7
+  - 3.8
+  - 3.9
+numpy:
```

Comment:
These python keys won't work. See the python spec in the conda-forge pinnings.

Reply:
Do they need to be like `3.6.*`?

Reply:
Nope. See here: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L602